ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -602,6 +607,11 @@ objects:
             exit 1
         fi
 
+        if [[ -n "${CLUSTER_NETWORK_MANIFEST}" ]]; then
+            openshift-install --dir=/tmp/artifacts/installer/create manifests
```

Comment:
This won't really work once we add more types of injection, and may not be how we decide to do this injection (profiles were supposed to be a config map of files injected into the pod that were then referenced by this job).

Please leave a `TODO: Replace with a more concise manifest injection approach`